modules/couchbase-cluster/variables.tf

```diff
@@ -42,72 +49,84 @@ variable "user_data" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "ssh_key_name" {
-  description = "The name of an EC2 Key Pair that can be used to SSH to the EC2 Instances in this cluster. Set to an empty string to not associate a Key Pair."
-  default     = ""
+  description = "The name of an EC2 Key Pair that can be used to SSH to the EC2 Instances in this cluster. Set to null to not associate a Key Pair."
+  type        = string
+  default     = null
 }
 
 variable "allowed_ssh_cidr_blocks" {
   description = "A list of CIDR-formatted IP address ranges from which the EC2 Instances will allow SSH connections"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "allowed_ssh_security_group_ids" {
   description = "A list of security group IDs from which the EC2 Instances will allow SSH connections"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "termination_policies" {
   description = "A list of policies to decide how the instances in the auto scale group should be terminated. The allowed values are OldestInstance, NewestInstance, OldestLaunchConfiguration, ClosestToNextInstanceHour, Default."
+  type        = string
   default     = "Default"
 }
 
 variable "associate_public_ip_address" {
   description = "If set to true, associate a public IP address with each EC2 Instance in the cluster."
+  type        = bool
   default     = false
 }
 
 variable "spot_price" {
   description = "The maximum hourly price to pay for EC2 Spot Instances."
+  type        = string
   default     = ""
```

Comment:
Set to `null`?

Reply:
Done: https://github.com/gruntwork-io/terraform-aws-couchbase/pull/49/commits/e49dc0e4d17bb77047e51f2a49916263cfc4afe1